From: Coly Li <colyli@suse.de>
Subject: kABI workaround for including mm.h in include/linux/net.h
Patch-mainline: Never, kABI workaround for linux-4.12 based SUSE kernel
References: bsc#1172873

patches.suse/0002-net-introduce-helper-sendpage_ok-in-include-linux-ne.patch
includes linux/mm.h into include/linux/net.h, which makes kABI checker warns
about kABI broken. This patch adds #ifndef __GENKSYMS__ to wrap the mm.h
including as a workaround of kABI warning.

Signed-off-by: Coly Li <colyli@suse.de>
---
 include/linux/net.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/net.h
+++ b/include/linux/net.h
@@ -26,7 +26,9 @@
 #include <linux/rcupdate.h>
 #include <linux/once.h>
 #include <linux/fs.h>
+#ifndef __GENKSYMS__
 #include <linux/mm.h>
+#endif
 
 #include <uapi/linux/net.h>
 
